src/config/proposals.json

```diff
@@ -2644,7 +2644,7 @@
     "cgpUrlRaw": "https://raw.githubusercontent.com/celo-org/governance/main/CGPs/cgp-0172.md",
```

Comment:
with the db does this file really need to exist at all?

Reply:
Not really